libs/googletest/Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY: all clean
+
+all: lib/libgtest.a
+
+lib/libgtest.a: lib/gtest-all.o
+	ar -rv lib/libgtest.a lib/gtest-all.o
+
+lib/gtest-all.o: lib/src/gtest-all.cc
+	g++ -I lib/include -I lib -c $< -o $@
+
+lib/src/gtest-all.cc: release-1.8.0.tar.gz
+	rm -rf lib
+	tar xfm release-1.8.0.tar.gz
```

Comment:
I suggest that we will have to update it manually. God knows what the new version would be like if once released, probably it would require altering the building commands and so on (just as it happened with move 1.6 -> 1.8, they added googlemock into the googletest repository and thus directory tree has significantly changed).